Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial
+
+# System dependencies
+RUN apt-get update && apt-get install --yes python3-pip
```

Comment:
I think that this should be updated on other projects, but it cleaner to clean the apt cache.
https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#run

``` Dockerfile
RUN apt-get update && apt-get install --yes python3-pip
 && rm -rf /var/lib/apt/lists/*
```

Reply:
replying to myself :smile: 

```
Note: The official Debian and Ubuntu images automatically run apt-get clean, so explicit invocation is not required.
```

Reply:
So does this mean no change is needed?

Reply:
it's not needed